CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 - [Integration](https://crates.io/crates/mcan-core) with the [`mcan`](https://crates.io/crates/mcan) crate.
 - Implementation of blocking::i2c::Transactional trait from [embedded-hal](https://crates.io/crates/embedded-hal) for TWI device.
 - Support for `critical-section` feature, falling down to PAC optional dependency.
+- Gitignore for `rust-analyzer.json` file used by the rustaceanvim Neovim plugin.
```

Comment:
My preference would be "Add `rust-analyzer.json` used by `rustaceanvin` Neovim's plugin to `.gitignore`"

Reply:
Changed to "Add `rust-analyzer.json` used by Neovim plugin `rustaceanvim` to `.gitignore`."